src/domain/resource/__tests__/utils.test.js

```diff
@@ -348,36 +346,14 @@ describe('domain resource utility function', () => {
 
     expect(resourceUtils.isTimeRangeReservable(resource, `${DATE}T08:00:00Z`, `${DATE}T10:00:00Z`))
       .toBe(true);
-    expect(resourceUtils.isTimeRangeReservable(resource, `${DATE}T08:00:00Z`, `${DATE}T16:00:00Z`))
-      .toBe(false);
+
     expect(resourceUtils.isTimeRangeReservable(resource, `${DATE}T08:00:00Z`, `${DATE}T16:00:00Z`, true))
```

Comment:
I assume `events`-argument should be something else than true. Maybe test events here?

Reply:
This test for made for `max_duration` check
Apparently there are no test for `events`, that's why i remove this test

Reply:
You are right, did not notice that the test was deleted 😄 All good.